includes/widgets/common.php

```diff
@@ -981,6 +981,461 @@ protected function register_controls() {
 			]
 		);
 
+		$this->start_controls_tabs( '_tabs_positioning' );
+
+		foreach ( [ 'normal', 'hover' ] as $tab ) {
+			$this->start_controls_tab(
+				"_tab_positioning_{$tab}",
+				[
+					'label' => 'normal' === $tab ? __( 'Normal', 'elementor' ) : __( 'Hover', 'elementor' ),
+				]
+			);
+
+			$this->add_control(
+				"_{$tab}_transform_rotate_popover",
+				[
+					'label' => __( 'Rotate', 'elementor' ),
+					'type' => Controls_Manager::POPOVER_TOGGLE,
+					'prefix_class' => 'e-',
```

Comment:
Shouldn't it be more specific? like: e-transform-rotate...

Reply:
The **'prefix_class' => 'e-'** and **'return_value' => 'transform',**:
Attached for each control which manipulate the element with CSS.
I need it for adding the class **"e-transform"** to the **"elementor-element" div.**
so I can select it by CSS 
![image](https://user-images.githubusercontent.com/79147990/124095634-87819a00-da62-11eb-91b3-93af8858d0e0.png)

Reply:
If all the values of the prefix_class and the return_value are exactly the same, maybe create 2 variables above the foreach?
1. $transform_prefix_class = 'e-';
2. $transform_return_value = 'transform';

Reply:
done